package.json

```diff
@@ -20,24 +20,26 @@
   },
   "dependencies": {
     "@bigcommerce/big-design": "^0.34.3",
-    "@bigcommerce/big-design-theme": "^0.17.0",
+    "@bigcommerce/big-design-icons": "^0.23.0",
+    "@bigcommerce/big-design-theme": "^0.18.0",
     "firebase": "^9.14.0",
     "jsonwebtoken": "^9.0.0",
     "mysql": "^2.18.1",
     "next": "^12.3.3",
     "node-bigcommerce": "github:bigcommerce/node-bigcommerce",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "styled-components": "^5.3.6",
-    "swr": "^0.5.7"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "styled-components": "^5.3.10",
+    "swr": "^2.1.5"
```

Comment:
🍹 I saw you likely updated `swr` because of the peer dependency error. Just something to keep in mind, it could be alarming for some to see a version go from `0.x.x` to `2.x.x` so suddenly, as that's two versions of potentially breaking changes that are being introduced. Just make sure to check that all the functions using `swr` are still functioning properly 👍 If anything's broken, it looks like the lowest version of SWR that supported React 18 was [1.1.0](https://github.com/vercel/swr/releases/tag/1.1.0), so you could use that version as a backup plan.

Reply:
Ahhh okay, thanks! Just updated that as well.

Reply:
Oh I didn't refresh the page and it stopped loading on the newer SWR version, something is going on with the way its consuming the JWT key on the newer version, the error isn't showing on the 1.1.0 though so I will leave it as that one.

Reply:
Awesome, great catch 👍